_posts/2019-05-28-bazel-0.26.0.md

```diff
@@ -0,0 +1,159 @@
+  updated to automatically fix this in your code.
+
+- The flag [`--incompatible_no_kwargs_in_build_files`](https://github.com/bazelbuild/bazel/issues/8021)
+  is added. It fixes a bug where the `**kwarg` notation was allowed in `BUILD`
+  files.
+
+- The flag [`--incompatible_static_name_resolution_in_build_files`](https://github.com/bazelbuild/bazel/issues/8022)
+  is added. This fixes a bug where invalid code was silently accepted.
+
+- The flag [`--incompatible_disallow_old_octal_notation`](https://github.com/bazelbuild/bazel/issues/8059)
+  is added. It removes a deprecated notation for octal numbers.
+  [Buildifier](https://github.com/bazelbuild/buildtools) can update your code
+  automatically.
+
+- [`struct.to_proto()`](https://docs.bazel.build/versions/master/skylark/lib/struct.html#to_proto)
+  can now convert dictionaries into proto3 text message.
+
+- The [`glob`](https://docs.bazel.build/versions/master/be/functions.html#glob)
+  function has a new argument `allow_empty`. When set to `False` (this will be
+  the default in the future), the glob fails when it doesn't match anything.
+  Please explicitly set `allow_empty = True` to the globs that need it.
+
+## Windows
+
+- Windows, Python: the new `--incompatible_windows_escape_python_args` flag
+  (`false` by default) builds `py_binary` and `py_test` targets with correct
+  command line argument escaping. Similar to
+  https://github.com/bazelbuild/bazel/issues/7486 but for `py_*` rules.
+
+## Other changes
+
+- cquery supports `--output=build`
+
+- Generated Go protobufs now depend on `//net/proto2/go:proto_gendeps` instead
+  of `//net/proto2/go:proto`
+
+- Added `--incompatible_remove_binary_profile` to disable the old binary
+  profiles. Instead use the JSON profile format:
+  https://docs.bazel.build/versions/master/skylark/performance.html#json-profile
+
+- Introducing `--execution_log_binary_file` and `--execution_log_json_file`
+  that output a stable sorted execution log. They will offer a stable
+  replacement to `--experimental_execution_log_file`.
+
+- New platform_mappings ability to allow gradual flag to platforms/toolchains
+  migration. See also https://github.com/bazelbuild/bazel/issues/6426
+
+- Added support for compiling against fully qualified `R` classes from
+  `aar_import` dependencies.
+
+- `--tls_enabled` flag is deprecated. Please provide `grpcs` as a scheme in
+  the URLs if TLS should be used for a remote connection.
+
+## Community
+
+- Alex Eagle gave a talk at ng-conf:
+  [The Bazel Opt-in Preview is Here!](https://www.youtube.com/watch?v=J1lnp-nU4wM)
+
+- The latest release of
+  [rules_haskell](https://github.com/tweag/rules_haskell/releases/tag/v0.9)
+  includes many new improvements, including an initial Windows support.
+
+- The latest
+  [Buildifier release](https://github.com/bazelbuild/buildtools/releases/tag/0.25.1)
+  includes Windows binaries, as well as a JSON output for diagnostics.
+
```

Comment:
Missing a note about VS Code:

- A new release of the [Visual Studio Code plugin for Bazel](https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel) was released.